packages/application-extension/src/index.tsx

```diff
@@ -552,6 +501,24 @@ const opener: JupyterFrontEndPlugin<void> = {
   },
 };
 
+/**
+ * TODO: remove, kept it for easier testing for now
```

Comment:
Kept for now so it's easier to test on ReadTheDocs and so the Pyodide kernel can enable file system access. Otherwise we need https://github.com/jupyterlite/pyodide-kernel/pull/178 to work with this PR.